estate_account/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+{
+    'name': 'Estate Account',
+    'version': '1.0',
+    'summary': 'Link module between Estate and Account modules',
+    'depends': [
+        'estate',
+        'account'
+        ],
+    'license': 'LGPL-3',
+    'installable': True,
+    'application': False,
+    'auto_install': False,
+    'data': [],
```

Comment:
```suggestion
    'data': [],
```
If it is empty then, there is no need to specify